dashboard/app/helpers/aichat_safety_helper.rb

```diff
@@ -24,16 +25,43 @@ def find_toxicity(role, text, locale)
         comprehend_response = AichatComprehendHelper.get_toxicity(text, locale)
         return {text: text, blocked_by: 'comprehend', details: comprehend_response} if comprehend_response && comprehend_response[:toxicity] > threshold
       end
+
+      if openai_enabled?(role)
```

Comment:
Should the openai check be first (or at least ahead of comprehend)?

Reply:
From a modeling perspective, I can't see wanting to use a mix of openai and comprehend, so I'm not sure it matters which is first. Arguably it should even be an error to have them both enabled, though I'm not sure its worth enforcing that.